src/XMakeBuildEngine/UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -3280,7 +3280,10 @@ public void Medley()
                 "$(e.Substring($([System.IO.Path]::Combine(`a`, `b`))))",
                 "$([]::())",
                 "$((((",
+#if FEATURE_WIN32_REGISTRY
+                // If no registry, this gets expanded to the empty string
                 "$(Registry:X)",
+#endif
```

Comment:
As is, today, it's not available in Core flavors at all, so I think this is the only way to go :(